src/main/java/microsoft/exchange/webservices/data/core/request/ServiceRequestBase.java

```diff
@@ -396,6 +396,39 @@ private static InputStream getResponseErrorStream(HttpWebRequest request)
   }
 
   /**
+   * Reads the response stream.
+   *
+   * @return serviceResponse
+   * @throws Exception
+   */
+  protected InputStream readResponseStream(HttpWebRequest response) throws Exception {
+    T serviceResponse;
+
+    if (!response.getResponseContentType().startsWith("text/xml")) {
+      String line = new BufferedReader(new InputStreamReader(ServiceRequestBase.getResponseStream(response)))
```

Comment:
Add comment that essentially says:
// We copied this from similar code upstream. It may serve as validation via side effects.